easybuild/toolchains/gcc.py

```diff
@@ -28,9 +28,10 @@
 @author: Kenneth Hoste (Ghent University)
 """
 
-from easybuild.toolchains.compiler.gcc import Gcc
+from easybuild.toolchains.gcccore import GCCcore
 
-
-class GccToolchain(Gcc):
+class GccToolchain(GCCcore):
     """Simple toolchain with just the GCC compilers."""
     NAME = 'GCC'
+    COMPILER_MODULE_NAME = [NAME]
+    SUBTOOLCHAIN = GCCcore.NAME
```

Comment:
having this here only makes sense if #1306 is merged (which we plan to do soon, so feel free to leave this here)